docs/source/app/phasechange.rst

```diff
@@ -0,0 +1,89 @@
+Phase-change materials
+======================
+Phase-change materials are some of the most promising materials for data-storage applications. In this application section, the pixelated STEM dataset with the AgInSbTe phase-change material specimen will be processed. Distinguishing between amorphous and crystalline regions can be hard in microstructure analysis with 4-D-STEM because of low contrast between them during the two most common method of visualization: bright and dark field imaging. The main difference of pixelated diffraction patterns for each of the beam positions the presence of additional (non zero-order) diffraction peaks in crystalline frames, while the amorphous frames have only the zero-order peak. Detection of the positions of additional peaks for all frames of crystalline regions will allow gaining information on the crystal structure.
+
+Next, will be described methods to distinguish crystalline and amorphous regions in phase-change materials.
+
+Crystallinity map
+~~~~~~~~~~~~~~~~~
+First, two different types of the regions, crystalline (low resistance) and amorphous (high resistance), should be distinguished. Crystalline regions are characterized by the presence of additional non-zero
+diffraction peaks on diffraction image, while the diffraction image (the frame) of amorphous regions contains only the zero-order diffraction peak.
```

Comment:
> "contains only the zero-order diffraction peak"

Written like this, it sounds like it doesn't contain anything else, but there may also be some diffuse scattering, right?